__init__.py

```diff
@@ -406,7 +402,7 @@ def handle_sensor_intent(self, message):
         # extract unit for correct pronounciation
         # this is fully optional
         try:
-            from quantulum import parser
+            from quantulum3 import parser
```

Comment:
Quantulum is an optional requirement on upstream, can we make an additional pull request for this in order to improve mycroft review procedure?
Or Is quantulum required to make the things work?

Reply:
I think it's required as without it mycroft complain wit error.

Reply:
Ok, in this case we can submit this as a whole fix. Maybe they'll ask for modifications in case they'll want to keep it an optional requirement